<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>511</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="welcomBanner">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome to StoBooks</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="sellerButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Seller</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buyerButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Buyer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPlainTextEdit" name="classNumBox">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="isbnBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="departmentLabel_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QRadioButton" name="okRB">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>250</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
     <widget class="QLabel" name="priceLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="isbnLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ISBN</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="priceBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="authorTextBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QRadioButton" name="likeNewRB">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>like new</string>
      </property>
     </widget>
     <widget class="QLabel" name="AuthorLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="goodRB">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>good</string>
      </property>
     </widget>
     <widget class="QLabel" name="classNumLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Class Number</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="editionBox">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>230</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="titleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
     <widget class="QLabel" name="conditionLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Condition</string>
      </property>
     </widget>
     <widget class="QLabel" name="editionLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edition</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="titleBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="departmentLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Department</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTextEdit" name="param1Box">
      <property name="geometry">
       <rect>
        <x>229</x>
        <y>30</y>
        <width>190</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="param2Box">
      <property name="geometry">
       <rect>
        <x>229</x>
        <y>120</y>
        <width>190</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="param3Box">
      <property name="geometry">
       <rect>
        <x>229</x>
        <y>210</y>
        <width>190</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="param1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>32</y>
        <width>178</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>choose parameter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Author</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Class Number</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Department</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Edition</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ISBN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Price</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Title</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="param2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>121</y>
        <width>178</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>choose parameter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Author</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Class Number</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Department</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Edition</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ISBN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Price</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Title</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>304</y>
        <width>202</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QComboBox" name="param3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>178</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Garamond</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>choose parameter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Author</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Class Number</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Department</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Edition</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ISBN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Price</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Title</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
     </font>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
     </font>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuwelcome_to_stoBooks">
    <property name="title">
     <string>welcome to stoBooks</string>
    </property>
   </widget>
   <addaction name="menuwelcome_to_stoBooks"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
